### PR TITLE
Added test for custom fscid cephfs

### PR DESCRIPTION
**Pull request by yogesh-mane**
# Description
Added test for recreating cephfs with custom fscid

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by yogesh-mane** [comment]
4.x logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UDT1YE
5.x logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-RL2ZQE

I will post full logs once run is completed

File: tests/cephfs/custom_fscid_cephfs.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            "ceph osd lspools | grep cephfs.cephfs",
+        ]
+        for command in verifyremove_command:
+            client1.exec_command(sudo=True, cmd=command, check_ec=False)
+            if client1.node.exit_status == 1:
+                log.info(f"{command} successfully executed")
+        commands = [
+            "ceph osd pool create cephfs_data 64 64",
+            "ceph osd pool create cephfs_metadata 8 8",
+            f"ceph fs new cephfs cephfs_metadata cephfs_data --fscid {fscid} --force",
```
Comment (AmarnatReddy) [inline tests/cephfs/custom_fscid_cephfs.py:58]: Hi Yogesh,
Can we create new file system with fscid. instead of deleting the existing and recreating it

Comment (yogesh-mane) [reply tests/cephfs/custom_fscid_cephfs.py]: Done Amar

**Comment by yogesh-mane** [comment]
4.x logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-OQYUG7
5.x logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-FOSZ1T

**Comment by yogesh-mane** [comment]
I will post full suite logs

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Comment by yogesh-mane** [comment]
Added "configure client" before test case for 5.x as full suite run failed 
5.x full run - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-VY530
4.x full run - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6T1YZ8

**Review by psathyan — APPROVED** [review]

**Pull request merged**